CHANGELOG-1.0.md

```diff
@@ -0,0 +1,121 @@
+# Release notes for master
```

Comment:
Can you put "1.0.0" at the top so it's more obvious which release this section corresponds to?

Reply:
done.